.github/workflows/build.yml

```diff
@@ -85,7 +89,7 @@ jobs:
         if: steps.s3-cache.outputs.processed == 'false'
         run: ${{ inputs.build_cmd }}
         env:
-          TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+          TURBO_TOKEN: ${{ inputs.turbo_cache && secrets.TURBO_TOKEN }}
```

Comment:
wouldn't this evalute to `false` and turbo try to use `false` as a token?

Reply:
Fixed, wasn't sure about the ternary. I'll try this out that it works as expected before merging